### PR TITLE
Updated default.properties so that pom.xml file becomes part of the g…

### PR DESCRIPTION
**Pull request by henrikengstrom**
…8 template.

### CODE REVIEW

File: project/default.properties [hunk 1]
```diff
@@ -1,2 +1,3 @@
-name=hello-akka-java
-description=This is a seed project and tutorial which creates a basic Java based Akka application.
+name = akka-quickstart-java
+description = Akka is a toolkit and runtime for building highly concurrent, distributed, and fault tolerant event-driven apps. This simple application will get you started building Actor based systems with Java.
+verbatim = *.xml
```
Comment (ktoso) [inline project/default.properties]: Ah, good to know - thanks! Likely the gradle file as to be included too?

Comment (henrikengstrom) [inline project/default.properties]: Actually, the `build.gradle` file is not filtered out by `g8` so there is no need for any specific setting for it.

File: project/default.properties [hunk 2]
```diff
@@ -1,2 +1,3 @@
-name=hello-akka-java
-description=This is a seed project and tutorial which creates a basic Java based Akka application.
+name = akka-quickstart-java
```
Comment (patriknw) [inline project/default.properties]: ok, this should be updated in the scala template also, I'll fix that

File: project/default.properties (hunk 1 above)
Comment (ktoso) [inline project/default.properties]: Ok cool, thanks for checking!

**Pull request merged**